### PR TITLE
Databases-week3

### PR DESCRIPTION
**Pull request by Yuliia-Balandiuk**

### CODE REVIEW

File: databases/week3/mealsharing_db.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+CREATE TABLE `review` (
+`id` INT(100) UNSIGNED NOT NULL AUTO_INCREMENT,
+`title` VARCHAR(255) NOT NULL,
+`description` TEXT NOT NULL,
+`meal_id` INT(100) UNSIGNED NOT NULL,
+`stars` INT(5) NULL,
+`created_date` DATE NOT NULL,
+PRIMARY KEY (`id`),
+CONSTRAINT `fk_meal` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+);
```
Comment (Elia-Git) [inline databases/week3/mealsharing_db.sql]: Good job.

File: databases/week3/queries.sql [hunk 2]
```diff
@@ -0,0 +1,69 @@
+SELECT *
+FROM `meal`;
```
Comment (Elia-Git) [inline databases/week3/queries.sql]: Why parenthesis around table name when quering ...? Remove all parenthesis where you use it around table name
for instance in lines. 2,6.8,12,15 etc

File: databases/week3/queries.sql [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+LIMIT 2;
+SELECT *
+FROM `meal`
+JOIN `review` ON `review`.`meal_id` = `meal`.`id`
+WHERE `stars` >= 4;
+SELECT *
+FROM `meal`
+INNER JOIN `reservation` ON `reservation`.`meal_id` = `meal`.`id`
+WHERE `meal`.`title` = 'Rack of lamb'
+ORDER BY `reservation`.`created_date`;
```
Comment (Elia-Git) [inline databases/week3/queries.sql]: Try this instead. I would also recommend you to include a line of what the task is (question for each solution). Here for instance
**-- Get reservations for a specific meal sorted by created_date**
```
SELECT meal.description,reservation.created_date AS reservation_Created_date
FROM meal
JOIN reservation ON meal.id = reservation.meal_id
where meal_id=3
ORDER BY reservation.created_date ASC;
```

**Review by Elia-Git — CHANGES_REQUESTED** [review]
Pls pay extra attention where you should use parenthesis and where you don't ...
For instance below code is correct but doesn't run in VS code as you use parenthesis where you shouldn't.

**SELECT  `meal`.`id`, `meal`.`title`, AVG(`review`.`stars`)
FROM `meal`
INNER JOIN `review` ON `meal`.`id` = `review`.`meal_id`
GROUP BY `meal_id`
ORDER BY AVG(`review`.`stars`) DESC;**

File: databases/week3/mealsharing_db.sql [hunk 4]
```diff
@@ -31,4 +31,20 @@ CREATE TABLE `review` (
 `created_date` DATE NOT NULL,
 PRIMARY KEY (`id`),
 CONSTRAINT `fk_meal` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
-);
\ No newline at end of file
+);
+INSERT INTO meal (id, title, description, location, `when`, max_reservations, price, created_date)
+VALUES (1, 'Crawfish soup', 'Traditional soup cooked on crawfish tails. Served with homemade dumplings with crayfish necks', 'Bregnegaardsvej 2A, 2920 Charlottenlund', '2022-08-01 10:00:00', 2, 250.50, '2022-08-01');
```
Comment (viedamayanti) [inline databases/week3/mealsharing_db.sql:36]: The id is auto increment, so you don't need to write it manually, sql will do it

**Pull request merged**